packages/fhir-location-management/src/components/EditLink/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Link, useLocation } from 'react-router-dom';
+import { useMls } from '../../mls';
+import {
+  BACK_SEARCH_PARAM,
+  URL_LOCATION_UNIT_EDIT,
+  URL_SERVICE_POINT_ADD_EDIT,
+} from '../../constants';
+import { ILocation } from '@smile-cdr/fhirts/dist/FHIR-R4/interfaces/ILocation';
+
+interface EditLinkProps {
+  location: ILocation;
+  editLinkText?: string;
+}
+
+const EditLink: React.FC<EditLinkProps> = (props) => {
+  const { location, editLinkText } = props;
+  const { t } = useMls();
+  const { id, physicalType } = location;
+  const isBuilding = physicalType?.coding?.[0].code === 'bu';
+  const currentLocation = useLocation();
+
+  const backToParam = new URLSearchParams({ [BACK_SEARCH_PARAM]: currentLocation.pathname });
+
+  return (
+    <Link
+      to={
+        isBuilding
+          ? `${URL_SERVICE_POINT_ADD_EDIT}/${id}`
```

Comment:
should `backToParam` be added to service point also?

Reply:
This isn't in place on the current view `https://web.unicef-mg-eusm-preview.smartregister.org/admin/service-points/add-edit/f15ff8ab-9475-4356-8363-7f518fdd66ce`.

Reply:
The bacToParam informs where we should navigate to on after a side effect on the form view. I think we should also add it for service points.